client/contract.go

```diff
@@ -13,14 +13,14 @@ const (
 // ContractClient is a client for the calling contract information
 type ContractClient struct {
 	ethClient ethclient.Client
-	cache     *cache.LRU
+	cache     *cache.LRU[common.Address, interface{}]
```

Comment:
I think this just stores common.Address -> `*ContractInfo`

Reply:
done. thanks